org.eclipse.gef/src/org/eclipse/gef/tools/AbstractTool.java

```diff
@@ -321,13 +326,9 @@ protected void applyProperty(Object key, Object value) {
 			}
 			if (property != null) {
 				Method setter = property.getWriteMethod();
-				// setter.setAccessible(true);
 				setter.invoke(this, value);
 			}
-		} catch (IntrospectionException ie) {
-		} catch (IllegalAccessException iae) {
```

Comment:
I usually like to catch a ReflectiveOperationException, rather than an IllegalAccessException, InvocationTargetException and SecurityException. What do you think?

Reply:
I'm normally not using reflection to much. Therefore I didn't check the exceptions. But yes your proposal is much cleaner. Changed it.